README.rst

```diff
@@ -90,10 +90,11 @@ Here's a full example with the libvirt provider:
          # use single quotes to avoid YAML parsing as dict due to ':'
          - 'vm.synced_folder ".", "/vagrant", type: "rsync"'
        # Dictionary of `config` options. Note that string values need to be
-       # explicitly enclosed in quotes.
+       # explicitly enclosed in quotes. To protect the quotes from being parsed
```

Comment:
That comment is misleading. I think we should not mention anything about quoting as that is up to YAML spec. 

Please remove the sentence about quoting.